### PR TITLE
ModelBuilderExtensions class implementation 😎

### PR DESCRIPTION
**Pull request by Evgeniy-Sinicin**

### CODE REVIEW

File: InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+            var userName3 = "Dasha";
+            var email3 = "user@example.com";
+            var userName4 = "Alexandr";
+            var email4 = "admin@gmail.com";
+
+            _users = new User[]
+            {
+                new User
+                {
+                    Id = "3827e25d-8989-4881-9f25-807d14878c1b",
```
Comment (romanstat) [inline InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs]: create guid

File: InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+    {
+        private static User[] _users;
+        private static Internship[] _internships;
+        private static Team[] _teams;
+        private static InternshipStack[] _internshipStacks;
+        private static Candidate[] _candidates;
+        private static Feedback[] _feedbacks;
+        private static Skill[] _skills;
+        private static Evaluation[] _evaluations;
+        private static InterviewInvite[] _interviewInvites;
```
Comment (romanstat) [inline InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs]: remove

File: InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+        {
+            builder.FillUsers();
+            builder.FillInternships();
+            builder.FillTeams();
+            builder.FillInternshipStacks();
+            builder.FillCandidates();
+            builder.FillFeedbacks();
+            builder.FillSkills();
+            builder.FillEvaluations();
+            builder.FillInterviewInvites();
```
Comment (romanstat) [inline InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs]: like linq extension method

File: InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+                {
+                    Id = "3827e25d-8989-4881-9f25-807d14878c1b",
+                    Position = "BA",
+                    BestContactTime = DateTime.UtcNow,
+                    UserName = userName1,
+                    NormalizedUserName = userName1.ToUpper(),
+                    Email = email1,
+                    NormalizedEmail = email1.ToUpper(),
+                    EmailConfirmed = false,
+                    PasswordHash = "AQAAAAEAACcQAAAAEFrfBeLl7wG60Syi2unYkVUM4kp/0A8RAZdc51M8e/5GokOv0qDZF+aGBSVaSV6vag==",
```
Comment (romanstat) [inline InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs]: new PasswordHash

File: InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,377 @@
...
+                    AccessFailedCount = 0
+                }
+            };
+
+            builder.Entity<User>().HasData(_users);
+        }
+
+        public static void FillInternships(this ModelBuilder builder)
+        {
+            var id = 1;
```
Comment (romanstat) [inline InternshipBE/DAL/Extensions/ModelBuilderExtensions.cs:214]: remove id, mssql autogenerate

File: InternshipBE/DAL/Entities/Candidate.cs [hunk 6]
```diff
@@ -51,5 +51,9 @@ public class Candidate
         public Team Team { get; set; }
 
         public ICollection<User> Users { get; set; }
+
+        public int InternshipId { get; set; }
+
+        //public int TeamId { get; set; }
```
Comment (romanstat) [inline InternshipBE/DAL/Entities/Candidate.cs]: remove comments

**Review by romanstat — APPROVED** [review]

**Review by Xodoc — APPROVED** [review]

**Review by vldmrklnk — APPROVED** [review]

**Pull request merged**